ide-lsp/src/main/java/org/aya/lsp/server/AyaLanguageServer.java

```diff
@@ -328,7 +328,7 @@ public Optional<SignatureHelp> signatureHelp(TextDocumentPositionParams params)
     var source = find(params.textDocument.uri);
     if (source == null) return Optional.empty();
     return Optional.of(FindReferences
-      .findRefs(source, libraries.view(), LspRange.pos(params.position))
+      .findRefsOutsideDefs(source, libraries.view(), LspRange.pos(params.position))
```

Comment:
This should be like before, recursive usages are still usages

Reply:
Then it would be strange: you clicked "1 usage" and found two usages

Reply:
Sounds acceptable to me

Reply:
Though I still feel strange...
I'm in bed now. You can push to this brance to revert the change.

Reply:
I'll leave it to you ;)

Reply:
We may change the message to "%d other usages".